ClientApp/src/components/reuseables/ProjectEvaluationResults/EvaluationNotesTable.tsx

```diff
@@ -23,20 +22,20 @@ export const EvaluationNotes: React.FC<EvaluationNotesProps> = ({
         </Row>
       </Card.Header>
       <ListGroup variant="flush">
-        {notes.map(
+        {notes?.map(
           ({
-            noteId: id,
-            note,
-            employeeReviewer: { firstName, lastName },
-          }: ProjectEvaluationEmployeeNote): JSX.Element => (
-            <ListGroup.Item key={id}>
+            content,
+            notesId,
+            employee: { fullName } = {},
```

Comment:
Object destructing is good for making code more readable, but when you start to destruct optional attributes the syntax gets a bit confusing. Personally I'd prefer
```suggestion
            employee,
```
and update line 34 to just use `employee?.fullName`.

Reply:
Sure thing. I kinda like the one-liners :P